test/GitcoinGovernor.t.sol

```diff
@@ -427,3 +428,466 @@ contract GitcoinGovernorAlphaPostProposalTest is GitcoinGovernorProposalTestHelp
     governorAlpha.queue(_newProposalId);
   }
 }
+
+contract NewGitcoinGovernorProposalTest is GitcoinGovernorProposalTestHelper {
+  // From GovernorCountingSimple
+  uint8 constant AGAINST = 0;
+  uint8 constant FOR = 1;
+  uint8 constant ABSTAIN = 2;
+
+  function assumeReceiver(address _receiver) public {
+    vm.assume(_receiver != TIMELOCK && _receiver != address(0x0));
+  }
```

Comment:
Why exclude the timelock and zero address here? Also maybe a more meaningful function that describes the assumption would be helpful

Reply:
Note added in https://github.com/gitcoinco/2022-Governor-upgrade/pull/12/commits/8d1ac767b13e9e5b51f451455cf48d15c844aa63